project/src/components/reviews-item/reviews-item.tsx

```diff
@@ -0,0 +1,38 @@
+import {TOTAL_STARS} from '../../const';
+import {ReviewType} from '../../types/mocksTypes';
+
+type ReviewPropsType = {
+  review: ReviewType;
+}
+function ReviewItem({review}: ReviewPropsType): JSX.Element {
+  const date = new Date(review.date);
+
+  return (
+    <li className="reviews__item">
+      <div className="reviews__user user">
+        <div className="reviews__avatar-wrapper user__avatar-wrapper">
+          <img className="reviews__avatar user__avatar" src={review.user.avatarUrl} width="54" height="54"
+            alt="Reviews avatar"
+          />
```

Comment:
если начала делать переносы, так переноси уже все. 
```suggestion
          <img 
            className="reviews__avatar user__avatar" 
            src={review.user.avatarUrl} 
            width="54"
            height="54"
            alt="Reviews avatar"
          />
```

Reply:
ну и кажется, что 54 стоит в константу вынести.